app/utilities/types.py

```diff
@@ -1,6 +1,9 @@
+from collections import namedtuple
 from typing import TypeAlias
 
 from app.questionnaire.location import Location
 from app.questionnaire.relationship_location import RelationshipLocation
 
 LocationType: TypeAlias = Location | RelationshipLocation
+
+SectionKey = namedtuple("SectionKey", "section_id list_item_id")
```

Comment:
I think the typed version of this would be even better (importing from typing)
```
class SectionKey(NamedTuple):
    section_id: str
    list_item_id: str | None
```

Reply:
Added 👍